plotly/offline/offline.py

```diff
@@ -50,16 +52,11 @@ def init_notebook_mode():
     yet. This is an idempotent method and can and should be called from any
     offline methods that require plotly.js to be loaded into the notebook dom.
     """
-    warnings.warn('''
-        `init_notebook_mode` is deprecated and will be removed in the
-        next release. Notebook mode is now automatically initialized when
-        notebook methods are invoked, so it is no
-        longer necessary to manually initialize.
-    ''', DeprecationWarning)
-
     if not _ipython_imported:
         raise ImportError('`iplot` can only run inside an IPython Notebook.')
 
+    global __PLOTLY_OFFLINE_INITIALIZED
+    # Inject plotly.js into the output cell
```

Comment:
I'm not actually, that's the heart of the fix. Before, we had this check:
```
if not __PLOTLY_OFFLINE_INITIALIZED:
    return
```

which meant that if you ran this function twice, we return `None` the second time which cleared the output cell (which previously had `plotly.js` in it). Now, we just inject `plotly.js` into the output cell every time.